src/index_create_aws_iam_roles.js

```diff
@@ -65,6 +67,9 @@ exports.handler = function(event, context, callback) {
       Object.assign(payload, options, roles[i]);
       payload.externalId = uuid.v4();
       payload = JSON.parse(JSON.stringify(payload));
+      if (payload.roleName == 'PowerUser') {
```

Comment:
Is this power user is required for gcloud also ? If so then we need to add it in the gcloud file as well.